libraries/standard/mqtt/lexicon.txt

```diff
@@ -0,0 +1,229 @@
+ack
+acked
+acks
+addrecord
+api
+app
+aws
+bufferlength
+bytesreceived
+bytestoread
+bytestoreceive
+bytestorecv
+bytestosend
+bytestowrite
+ca
+calculatestatepublish
+cbmc
+cleansession
+cmock
+com
+config
+connack
+connectinfo
+connectpacketsize
+const
+createcleansession
+currentstate
+defragmenting
+deserialization
+deserialize
+deserializeack
+deserialized
+deserializepublish
+deserializing
+didn
```

Comment:
The part before the apostrophe in contractions like *hasn't*, *doesn't* and *didn't* are flagged by the script and need to be part of the lexicon.

Reply:
Was this the problem for v4_beta? How did we handle it there ?

Reply:
This was the same behavior in `v4_beta` and we used the same solution in the lexicon files there.